### PR TITLE
Drop legacy dbs, add optional phpMyAdmin, start deprecation of networks.

### PR DESCRIPTION
**Pull request by iBotPeaches**
* I don't believe any projects is using 56 anymore, if so please PM me.
* Maria102 is a year+ away from going EOL, so we need to add the next major for project opt-in.
* New DB leverages our internal "st-internal" network so we don't have an n+1 problem with network creation.

Screen Shot 2020-03-05 at 9 38 41 AM

 * We can now support multiple hosts via one management system.
 * Start deprecation of specific service networks for one

---
CHANGES
 * Dropped mysql56
 * Added mariadb103
 * Added mariadb104
 * Added optional phpmyadmin (hooks to all 3 marias)
 * Updated examples for Yii2/Laravel, updating PHP
 * Dropped network creation (deprecation)
 * Kept old networks for backward compatibility

---
IF MERGED

 * Projects should opt into `st-internal` for service specific things before the network dies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,10 +24,9 @@ of these services, you likely would not use a docker version of them in producti
 - If this fails, then manual configuration is necessary before compilation. Perform the following:
    1. `docker network create nginx-proxy`
    2. `docker network create st-mariadb-102`
```
Comment (jweberst) [inline README.md]: Are allt hese networks unneeded now?

Comment (iBotPeaches) [reply README.md]: For new dbs yes, I could do a large breaking change and opt all the others to internal but it would require project updates for all.

Comment (jweberst) [reply README.md]: Maybe this release should add them both networks now, and we can tell people to update the projects... then, the next change, remove the networks. Something can be on two networks, right?

Comment (iBotPeaches) [reply README.md]: That does work. I've introduced this here to review separately (along w/ feature) - https://github.com/sourcetoad/DevopsToolKit/pull/11

I can then merge it into this.

**Comment by ToothlessRebel** [comment]
I recently got IQWST running locally via Docker on Mysql56. I would assume it could run on the MariaDB server but I haven't checked yet.

**Pull request merged**